Forge.TreeWalker/src/TreeWalkerParameters.cs

```diff
@@ -132,6 +132,11 @@ public class TreeWalkerParameters
         /// </summary>
         public object TreeInput { get; set; }
 
+        /// <summary>
+        /// Flag that will allow forge to re-execute the Node upon re-hydration.
```

Comment:
>Flag that will allow forge to re-execute the Node upon re-hydration [](http://example.com/codeflow?start=12&length=67)

Flag that allows Forge to re-execute the current TreeNode's Actions upon rehydration.